src/RelaxedServiceProvider.php

```diff
@@ -17,6 +17,10 @@
 class RelaxedServiceProvider extends ServiceProviderBase {
 
   public function alter(ContainerBuilder $container) {
+    // Override the access_check.permission class with a new class.
+    $definition = $container->getDefinition('access_check.permission');
+    $definition->setClass('Drupal\relaxed\Access\PermissionAccessCheck');
```

Comment:
Instead of overriding the `access_check.permission` service, I think it's better to use the existing conjunction functionality that's already implemented in this core service.

So we should be able to add "+perform content replication" to `_permission` in `\Drupal\relaxed\Plugin\rest\resource\ResourceBase::routes`.

This way we don't need to override any core services.